ios/fastlane/Matchfile

```diff
@@ -0,0 +1,14 @@
+git_url("git@github.com:digitalfabrik/lunes-credentials.git")
+storage_mode("git")
+
+username("delivery@lunes.app")
```

Comment:
From where is this username? Is it a apple account name? If yes, then I think there is no more a need for a special apple account.

We are using now api keys.

Reply:
It is the github-user, which has the ssh key for accessing the credentials repository.